static/css/restyle.less

```diff
@@ -28,6 +29,17 @@
     src: local('Open Sans Italic'), local('OpenSans-Italic'), url('../../css/impala/fonts/OpenSans-Italic.ttf') format('ttf');
 }
 
+// Mixins
+.drop-shadow {
```

Comment:
Do you think this will be re-used? I'm not sure the mixin is necessary. Also having the `&:after` in it seems a bit odd to me.